### PR TITLE
Change __init__ files

### PR DESCRIPTION
**Pull request by GuyPuts**

### CODE REVIEW

File: opensquirrel/__init__.py [hunk 1]
```diff
@@ -1,4 +1,73 @@
...
+    barrier,
+    default_bloch_sphere_rotation_set,
+    default_bloch_sphere_rotation_without_params_set,
+    default_unitary_set,
+    init,
+    measure,
+    mX90,
+    mY90,
+    reset,
+    wait,
```
Comment (rturrado) [inline opensquirrel/__init__.py:33]: Do we want to sort these lines?

File: opensquirrel/__init__.py [hunk 2]
```diff
@@ -1,4 +1,73 @@
...
+    "CRk",
+    "SWAP",
+    "measure",
+    "init",
+    "reset",
+    "barrier",
+    "wait",
+    "default_bloch_sphere_rotation_without_params_set",
+    "default_bloch_sphere_rotation_set",
+    "default_unitary_set",
```
Comment (rturrado) [inline opensquirrel/__init__.py:72]: Do we want to sort these lines?

File: opensquirrel/__init__.py [hunk 3]
```diff
@@ -1,4 +1,73 @@
...
+    Sdag,
+    T,
+    Tdag,
+    X,
+    Y,
+    Z,
+    barrier,
+    default_bloch_sphere_rotation_set,
+    default_bloch_sphere_rotation_without_params_set,
+    default_unitary_set,
```
Comment (rturrado) [inline opensquirrel/__init__.py:27]: I would take these out. They are internal variables, not something a user would need to easily access.

File: opensquirrel/__init__.py (hunk 2 above)
Comment (rturrado) [inline opensquirrel/__init__.py:72]: I think we can take these out.

File: opensquirrel/passes/merger/general_merger.py [hunk 4]
```diff
@@ -6,10 +6,10 @@
 
 import numpy as np
 
+from opensquirrel import default_bloch_sphere_rotation_without_params_set
```
Comment (rturrado) [inline opensquirrel/passes/merger/general_merger.py:9]: This line should stay as it was.

File: opensquirrel/utils/__init__.py [hunk 5]
```diff
@@ -1,6 +1,18 @@
...
+    "are_axes_consecutive",
+    "filter_out_identities",
+    "flatten_list",
+    "get_matrix",
+    "check_mapper",
+    "expand_ket",
+    "get_reduced_ket",
+    "can1",
+    "get_matrix",
+]
```
Comment (rturrado) [inline opensquirrel/utils/__init__.py:18]: Nice! I think, even these identifiers being internal, we want to have them in this init file for our own sake (to make their referencing easier from everywhere).

Comment (rturrado) [reply opensquirrel/utils/__init__.py:18]: Have we checked also that, everywhere in the code, we just do `from opensquirrel.utils import...` and not `from opensquirrel.utils.blah import...`?

File: opensquirrel/__init__.py [hunk 6]
```diff
@@ -1,4 +1,73 @@
...
+from opensquirrel.writer import writer
 
-__all__ = ["Circuit", "CircuitBuilder"]
+__all__ = [
+    "Circuit",
+    "CircuitBuilder",
+    "writer",
+    "InverseGateModifier",
+    "PowerGateModifier",
+    "ControlGateModifier",
```
Comment (rturrado) [inline opensquirrel/__init__.py:43]: I think we can take these out.

**Review by rturrado — COMMENTED** [review]
Beautiful, I like it!

I've pointed out a few items I would exclude.

File: opensquirrel/__init__.py (hunk 1 above)
Comment (GuyPuts) [reply opensquirrel/__init__.py:33]: Regarding this, would it be a good idea to run isort on the whole library? There are more unsorted imports in multiple files; isort would sort and format them alphabetically

File: opensquirrel/utils/__init__.py (hunk 5 above)
Comment (GuyPuts) [reply opensquirrel/utils/__init__.py:18]: I've checked it for every import that I added to init. I'll double check just to be sure.

**Review by rturrado — APPROVED** [review]
It looks good to me.

But let's anyway wait for Chris to do his review.

Thanks Guy!

**Review by elenbaasc — APPROVED** [review]
Looks all good, thanks for the effort!

**Pull request merged**